packaging/msi/templates/choco/LICENSE.txt

```diff
@@ -0,0 +1,9 @@
+From: {{.Choco.LicenseURL}}
```

Comment:
Thinking a bit about this, for now it's probably easiest just to get the wix support into the build pipeline so we're producing and uploading a real msi installer, then come back to thinking about Choco support.

Thoughts?

Reply:
Sounds good, I'll remove the Chocolatey artifacts for now.